src/screens/WalletScreen/SendMoney/index.js

```diff
@@ -167,7 +167,7 @@ class SendMoney extends NavigatorComponent<Props, State> {
                         placeholderTextColor={Colors.placeholderTextColor}
                         onChangeText={amountString => this.setState({ amountString })}
                         value={this.state.amountString}
-                        keyboardType='decimal-pad'
```

Comment:
As you specified on PR description:
`Fixed keyboards for numbers are still alphanumeric, they should be numeric keypads.`
Does it mean that `decimal-pad` set before produce alphanumeric keyboard? I believe we have changed it from numeric to decimal-pad intentionally so that it works consistent between iOS and Android. So, can you please specify a bit more, what exactly are you trying to fix (screenshot of previous incorrect behavior would be great).

Reply:
@seland I think this PR need to be postponed because I have fixed it on the android keyboard, before the fix it will be the alphanumeric keyboard as you see in the photo below. But because of the issue facebook/react-native#17473, I'm not potential it could fix for all android devices. So we need more research and testing on this case
![screen shot 2018-10-31 at 8 06 06 pm](https://user-images.githubusercontent.com/1697756/47789901-71b6f680-dd48-11e8-903f-c4827010020a.png)